src/controllers/User.ts

```diff
@@ -77,7 +79,7 @@ export async function update(req: Request, res: Response) {
         }
 
         // Verify old password or comparing password
-        const isOldPasswordValid = await bcrypt.compare(oldPassword, oldUser.password);
+        const isOldPasswordValid = await bcrypt.compare(oldPassword ?? "", oldUser.password);
         if (!isOldPasswordValid) {
             res.status(400).json(errorJson("Old password doesn't match", null));
             return;
```

Comment:
Need to fix but, you do not need to do anything. Just remind me at the meeting to explain you why does this needed to be fixed.